src/Main.elm

```diff
@@ -38,24 +97,19 @@ update _ model =
 
 header : String -> Html msg
 header text =
-    Html.span [ Attrs.class "p-2 text-5xl font-extrabold text-transparent bg-clip-text bg-gradient-to-br from-slate-400 to-slate-800" ]
-        [ Html.text text ]
-
-
-subheader : String -> Html msg
-subheader text =
-    Html.span [ Attrs.class "p-2 text-2xl font-extrabold text-slate-800" ]
-        [ Html.text text ]
+    Html.div
+        [ Attrs.class "p-2 text-5xl font-extrabold text-transparent bg-clip-text bg-gradient-to-br from-slate-400 to-slate-800"
+        , A11y.label text
+        ]
+        [ Html.text text
+        ]
 
 
 view : Model -> Html Msg
-view _ =
-    Html.div [ Attrs.class "flex flex-col w-[1024px] items-center mx-auto mt-16 mb-48" ]
-        [ header "Let's start your task"
-        , subheader "Here are your data:"
-        , Html.pre [ Attrs.class "my-8 py-4 px-12 text-sm bg-slate-100 font-mono shadow rounded" ] [ Html.text Data.userGroup ]
-        , header "Now turn them into form."
-        , subheader "See README for details of the task. Good luck 🍀 "
+view model =
+    Html.div [ Attrs.class "flex flex-col items-center mx-auto mt-16 mb-48" ]
```

Comment:
The form looks really nice, thanks for using tailwind! ❤️

Reply:
My absolute pleasue, been a long time fan of Tailwind myself, it's very helpful albeit a bit verbose